input/intro-notes/StructureDefinition-au-core-practitioner-intro.md

```diff
@@ -32,4 +32,8 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
     </p>
 </div>
 
-<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the mandatory requirement to support _revinclude of Provenance.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43850">Jira issue #FHIR-43850</a>.</p>
\ No newline at end of file
+<p class="request-for-feedback">Specific feedback is sought on the proposal to remove the mandatory requirement to support _revinclude of Provenance.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43850">Jira issue #FHIR-43850</a>.</p>
+
+<p class="request-for-feedback">Specific feedback is sought on the inclusion of addresses and telecom in the Practitioner's profile.<br/>Please comment on <a href="https://jira.hl7.org/browse/FHIR-43860">Jira issue #FHIR-43860</a>.</p>
```

Comment:
Replace
Specific feedback is sought on the inclusion of addresses and telecom in the Practitioner's profile.

With
Specific feedback is sought on removing Must Support from Practitioner.addresses and Practitioner.telecom.

Reply:
Done

Reply:
Applied change, and moved to the bullet list with other remove Must Support proposals.